src/components/Viewer/InformationPanel/Annotation/VTT/Cue.tsx

```diff
@@ -1,16 +1,45 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useRef, useState } from "react";
 
+import { AutoScrollOptions } from "src/context/viewer-context";
 import { Item } from "src/components/Viewer/InformationPanel/Annotation/VTT/Cue.styled";
 import { convertTime } from "src/lib/utils";
+import {
+  ViewerContextStore,
+  useViewerDispatch,
+  useViewerState,
+} from "src/context/viewer-context";
+
+const AutoScrollDisableTime = 750;
 
 interface Props {
   label: string;
   start: number;
   end: number;
+  autoScroll?: AutoScrollOptions;
 }
 
+const findScrollableParent = (
+  element: HTMLElement | null,
+): HTMLElement | null => {
+  while (element && element !== document.body) {
+    const overflowY = window.getComputedStyle(element).overflowY;
+    const isScrollable = overflowY !== "visible" && overflowY !== "hidden";
+
+    if (isScrollable && element.scrollHeight > element.clientHeight) {
+      return element;
+    }
+
+    element = element.parentNode as HTMLElement;
+  }
+
+  return null;
+};
+
 const Cue: React.FC<Props> = ({ label, start, end }) => {
+  const dispatch: any = useViewerDispatch();
+  const { autoScroll, userScrolling }: ViewerContextStore = useViewerState();
```

Comment:
I'm not sure exactly what's going on, it looks like this isn't peeling off the autoScroll correctly, at least in my local. For it to work for me, I need to dig into `configOptions.informationPanel.vtt.autoScroll`.